platform/source/minimal-printf/mbed_printf_implementation.c

```diff
@@ -130,105 +140,104 @@ static void mbed_minimal_putchar(char *buffer, size_t length, int *result, char
 }
 
 /**
- * @brief      Print signed integer.
+ * @brief      Print integer in signed, unsigned or hexadecimal format.
  *
- * @param      buffer  The buffer to store output (NULL for stdout).
- * @param[in]  length  The length of the buffer.
- * @param      result  The current output location.
- * @param[in]  value   The value to be printed.
+ * @param      buffer           The buffer to store output (NULL for stdout).
+ * @param[in]  length           The length of the buffer.
+ * @param      result           The current output location.
+ * @param[in]  value            The value to be printed.
+ * @param      type             The type of integer format that shall be printed (signed, unsigend or hexadecimal)
+ * @param      width_size       The width modifier.
+ * @param      prepend_zeros    Flag to prepends zeros when the width_size is greater than 0
  */
-static void mbed_minimal_formatted_string_signed(char *buffer, size_t length, int *result, MBED_SIGNED_STORAGE value, FILE *stream)
+static void mbed_minimal_formatted_string_integer(
+    char *buffer,
+    size_t length,
+    int *result,
+    MBED_UNSIGNED_STORAGE value,
+    integer_type_t type,
+    int width_size,
+    bool prepend_zeros,
+    FILE *stream)
```

Comment:
Ah, this got missed. Could you put back all the arguments in one line for consistency?

Reply:
Yeah sure! Sorry I just missed this ;-).